fixtures/webstudio-remix-netlify-functions/app/extension.ts

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
Why ignore?

Reply:
Import count as not used, but its needed